### PR TITLE
Add Web worker talk

### PR DESCRIPTION
**Pull request by mstruebing**

### CODE REVIEW

File: decks/web-worker.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+https://farbenmeer.github.io/ausguck/examples/web-worker/index.html
+
+---
+
+## Resources
+
+* https://en.wikipedia.org/wiki/Web_worker
+* https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API
+* https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers
```
Comment (Kampfheizung) [inline decks/web-worker.mdx]: I think you need to have one br after and before each `---`

File: examples/web-worker/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>stuff</title>
```
Comment (Kampfheizung) [inline examples/web-worker/index.html]: You could rephrase it to "web-worker" :P

File: decks/web-worker.mdx (hunk 1 above)
Comment (mstruebing) [reply decks/web-worker.mdx]: Thank you, fixed :)

File: examples/web-worker/index.html (hunk 2 above)
Comment (mstruebing) [reply examples/web-worker/index.html]: Renamed to `Web Worker` :+1:
